### PR TITLE
Add RSVP Badge

### LINKED ISSUE #185: Add RSVP counters to events
The API has RSVP data for FB events
We can display it in list view like this: 
screenshot 2017-07-12 11 29 29

And also on event details pages like this:
screenshot 2017-07-12 11 44 06

### PR DESCRIPTION
**Pull request by ignu**
Fixes #185 

Event List:

pasted image

Event Details:

pasted image

### CODE REVIEW

File: src/components/EventCard/EventCard.test.js [hunk 1]
```diff
@@ -27,18 +27,12 @@ describe('Component: EventCard', () => {
...
   });
 
   it('displays the correct time format for multi-day event', () => {
     props.event.end_date = '2017-04-30T21:00:00-07:00';
     const wrapper = shallow(<EventCard {...props} />);
-    const dateDisplay = wrapper.find('div').at(1).children('div').at(1)
-    .text()
-    .toLowerCase();
-    expect(dateDisplay).toBe('6:00 pm apr 29 - 9:00 pm apr 30');
+    expect(wrapper.text().toLowerCase()).toMatch('6:00 pm apr 29 - 9:00 pm apr 30');
```
Comment (dorono) [inline src/components/EventCard/EventCard.test.js]: Nice! I think that was my janky selector, me = still a Jest n00b :)

File: src/components/RsvpBadge/RsvpBadge.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import React, { PropTypes } from 'react';
+import styles from './RsvpBadge.sass';
+
+const toK = (n) => {
+  return n >= 1000 ? (n / 1000).toFixed(1) + 'K' : n;
```
Comment (pdw207) [inline src/components/RsvpBadge/RsvpBadge.js]: I think `1100K` could be 1.1 million. Lets think big.

File: src/components/EventCard/EventCard.js [hunk 3]
```diff
@@ -37,20 +38,24 @@ const EventCard = ({ event, className }) => {
...
+            }}
+          >
+            <img
+              src={featuredImageUrlOrDefault}
+              alt="featured event"
+            />
+          </Link>
+        </div>
+
+        <RsvpBadge totalAccepted={totalAccepted} center />
```
Comment (pdw207) [inline src/components/EventCard/EventCard.js]: Should we have logic to display/hide if totalAccepted is not provided?

**Comment by pdw207** [comment]
👍

**Pull request merged**